src/Jackalope/Transport/DoctrineDBAL/Query/QOMWalker.php

```diff
@@ -841,6 +851,38 @@ private function sqlXpathExtractValue(string $alias, string $property, string $c
         throw new NotImplementedException(sprintf("Xpath evaluations cannot be executed with '%s' yet.", $this->platform->getName()));
     }
 
+    private function sqlXpathExtractValueWithNullProperty(string $alias, string $column = 'props'): string
```

Comment:
does it make sense to have a parameter with default value for $column? if i understand correctly, this parameter is never changed, so we should use 'props' directly - or set up a constant in the class.

Reply:
@dbu I think you're right, we could remove $column. When I look at the method above, it's hard-coded (https://github.com/jackalope/jackalope-doctrine-dbal/blob/8d0ad925f99792037884cafa08bfd8c7cf4cc994/src/Jackalope/Transport/DoctrineDBAL/Query/QOMWalker.php#L886). I can do the same thing here. But what impact will that have? It's BC, isn't it?

Reply:
thanks. the `sqlXpathExtractValue` with `numerical_props` as column name in one case, so we can't remove it. for your new method we don't need it. (not quite sure if this means full text search might do something funny on numbers, but don't think we can do much about that)